broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/SnapshotReplicationPartitionBootstrapStep.java

```diff
@@ -8,18 +8,18 @@
 package io.camunda.zeebe.broker.system.partitions.impl.steps;
 
 import io.camunda.zeebe.broker.Loggers;
-import io.camunda.zeebe.broker.system.partitions.PartitionBoostrapAndTransitionContextImpl;
-import io.camunda.zeebe.broker.system.partitions.PartitionStep;
+import io.camunda.zeebe.broker.system.partitions.PartitionBootstrapContext;
+import io.camunda.zeebe.broker.system.partitions.PartitionBootstrapStep;
 import io.camunda.zeebe.broker.system.partitions.SnapshotReplication;
 import io.camunda.zeebe.broker.system.partitions.impl.NoneSnapshotReplication;
 import io.camunda.zeebe.broker.system.partitions.impl.StateReplication;
 import io.camunda.zeebe.util.sched.future.ActorFuture;
 import io.camunda.zeebe.util.sched.future.CompletableActorFuture;
 
-public class SnapshotReplicationPartitionStep implements PartitionStep {
+public class SnapshotReplicationPartitionBootstrapStep implements PartitionBootstrapStep {
 
   @Override
-  public ActorFuture<Void> open(final PartitionBoostrapAndTransitionContextImpl context) {
```

Comment:
I assume we should then complete the future with the context ? :thinking:

Reply:
Yes, that's the idea for the partition bootstrap steps.
For the partition transition steps it's still a void, because the context has only getters, so there is nothing to update in the context